test/integration/webhooks/leaderworkerset_test.go

```diff
@@ -261,6 +261,24 @@ var _ = ginkgo.Describe("leaderworkerset defaulting, creation and update", func(
 			},
 			lwsCreationShouldFail: true,
 		}),
+		ginkgo.Entry("update to size when resizePolicy is None should fail", &testValidationCase{
```

Comment:
This is only testing the validation logic. Can we add an integration test to verify that the controller actually applies this change to the statefulsets?